src/OmniSharp.Abstractions/Models/Events/EventTypes.cs

```diff
@@ -2,6 +2,8 @@ namespace OmniSharp.Models.Events
 {
     public static class EventTypes
     {
+        public const string ProjectLoadingStarted = nameof(ProjectLoadingStarted);
```

Comment:
Does this event mean that we have started loading all projects or that we have started loading a single project? If it means a single project then I would recommend `ProjectLoadStarted`. Same with the Finished event.

Reply:
Single one.
Done.